File: AICareerBuddy-Entities/Entities/JobListing.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AICareerBuddy_Entities.Entities
{
    public class JobListing
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Category { get; set; }
        public string Location { get; set; }
        public DateTime ListingExpires { get; set; }
        public List<string> Terms { get; set; }
    }
}

File: app/src/main/java/com/example/myapplication/models/JobListing.kt
package com.example.myapplication.models
import java.time.LocalDate

data class JobListing(
    val name: String,
    val description: String,
    val category: String,
    val location: String,
    val listingExpires: LocalDate,
    val terms: String,
    val payPerHour: Int,
    val employerId: Int
)